posts/inside-rust/2019-12-18-bisecting-rust-compiler.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "Bisecting Rust Compiler Regressions with cargo-bisect-rustc"
+author: Santiago Pastorino
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Let's say that you've just updated the Rust compiler version and have
+tried to compile your application and see a failure that wasn't there
+before. That's likely due to a regression in the compiler.  We've just
+released
+[`cargo-bisect-rustc`](https://github.com/rust-lang/cargo-bisect-rustc),
+this tool makes it super easy to find exactly when behavior has
+regressed in rustc. It automatically downloads rustc artifacts and tests
+them against a project you provide until it finds the regression. At
+minimum, it will identify the nightly release which triggered the
+regression; but if the regression occurred in the last 90 days, it will
+even figure out the exact PR, which is often very useful in helping us
+fix the problem. It was created originally by Mark Rousskov and I was
```

Comment:
I would make a new paragraph here, but also this sentence and the one after seem to say the same things.